packages/data-fetcher/src/utils/token.ts

```diff
@@ -0,0 +1,6 @@
+import { utils } from "zksync-web3";
+export const isNativeToken = (tokenAddress: string): boolean => {
+  const isL1ERC20 = tokenAddress.toLowerCase() === utils.L2_ETH_TOKEN_ADDRESS.toLowerCase();
```

Comment:
Why doesn't it compare to Base token address?

Reply:
Now it does.